modules/calib3d/src/usac/quality.cpp

```diff
@@ -421,7 +421,11 @@ class SPRTImpl : public SPRT {
      * @current_hypothesis: current RANSAC iteration
      * Return: true if model is good, false - otherwise.
      */
-    inline bool isModelGood (const Mat &model) override {
+    inline bool isModelGood(const Mat& model) override
+    {
+        if (model.empty())
+            return false;
```

Comment:
@ivashmak @vpisarev Is it valid?

Reply:
Yes it is. However, the situation that model comes empty to SPRT should not have happened in general. Maybe there is a problem somewhere else, I'll check it. For function setModelParameters CV_CheckTypeEQ(model.depth(), CV_64F, ""); it is a bit redundant because the model will always be in double format. It comes from solver.